app/src/app/components/contents/PacketGroup/PacketGroup.tsx

```diff
@@ -1,19 +1,48 @@
-import { useParams } from "react-router-dom";
 import { PacketTable } from "./PacketTable";
+import {useParams} from "react-router-dom";
+import {useGetPacketGroupSummaries} from "../explorer/hooks/useGetPacketGroupSummaries";
+import {HttpStatus} from "../../../../lib/types/HttpStatus";
+import {Unauthorized} from "../common/Unauthorized";
+import {ErrorComponent} from "../common/ErrorComponent";
+import {useGetPacketById} from "../common/hooks/useGetPacketById";
 
 export const PacketGroup = () => {
   const { packetName } = useParams();
+  const { packetGroupSummaries, error: packetGroupError } =
```

Comment:
there are now 3 network requests here including in Packettable... surely it can all be combined into 1... does pulling the one from packettable work? that should get you all the data you need

Reply:
I ended up implementing a new endpoint that would give us everything needed by the PacketGroup component: given a packet group name, return the display name and the latest packet's description. So now this only makes one network call! (Except PacketTable still makes a separate call for different info)